### PR TITLE
Support for setuptools_scm

### LINKED ISSUE #91: Add setuptools_scm for project
Make life a little better (manage project versions by scm tags)

https://github.com/pypa/setuptools_scm

### PR DESCRIPTION
**Pull request by pythad**
Added support for [setuptools_scm](https://github.com/pypa/setuptools_scm). Closes #91

### CODE REVIEW

**Comment by webknjaz** [comment]
@pythad better use imperative-style commit titles.

File: setup.py [hunk 1]
```diff
@@ -30,22 +28,9 @@ def package_files(directory):
...
-
-    with open(init_py) as f:
-        for line in f:
-            match = REGEXP.match(line)
-            if match is not None:
-                return match.group(1)
-        else:
-            msg = f'Cannot find version in ${init_py}'
-            raise RuntimeError(msg)
+setup_requires = [
```
Comment (webknjaz) [inline setup.py]: Also better add `pyproject.toml` with that as well.

File: create_aio_app/__init__.py [hunk 2]
```diff
@@ -1 +1,7 @@
-__version__ = '0.0.6'
+from pkg_resources import get_distribution, DistributionNotFound
```
Comment (webknjaz) [inline create_aio_app/__init__.py]: Plz add a module docstring as per PEP 257

**Comment by webknjaz** [comment]
@Arfey FTR you could set up pushes to Test PyPI on each commit if it's extended like in https://github.com/sanitizers/octomachinery/blob/master/octomachinery/utils/versiontools.py#L9-L38

**Comment by webknjaz** [comment]
@pythad please use `closes #91` in the PR description as per https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by pythad** [comment]
Ahhhhh, @webknjaz, enjoying these Linus style reviews!

ok, I'll do appropriate changes.

**Comment by webknjaz** [comment]
@pythad it's okay to do improvements as separate PRs, though. Changes should be atomic after all.

Here's some materials on writing commit messages:

                                                 

According to common commit style guides, it is required to write messages in an imperative manner. Make them actionable. Also **avoid using `-m`** as you often need to express more detail in a long way. Here's several articles sharing best practices:
* [ ] http://chris.beams.io/posts/git-commit/
* [ ] http://alistapart.com/article/the-art-of-the-commit
* [ ] https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message
* [ ] https://github.com/erlang/otp/wiki/writing-good-commit-messages
* [ ] https://wiki.openstack.org/wiki/GitCommitMessages
* [ ] http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
* [ ] https://dev.to/gonedark/when-to-make-a-git-commit

**Comment by Arfey** [comment]
thx 👍

**Pull request merged**